src/commands/mod.rs

```diff
@@ -59,6 +59,16 @@ pub enum Commands {
         #[arg(short, long)]
         position: Option<String>,
     },
+    /// Show a diff of operations and render the consolidated graph
+    #[command(name = "view-diff", arg_required_else_help(true))]
```

Comment:
follow up to make the current diff subarg this and put that command somewhere else